languages.js

```diff
@@ -110,6 +114,8 @@ var abbreviations = {
     'hu': abbreviationsHu,
     'lt': abbreviationsLt,
     'nl': abbreviationsNl,
+    'pt-BR': abbreviationsPt,
```

Comment:
And maybe also item for "generic" Portuguese `'pt': abbreviationsPt,`?

Reply:
This PR copies the abbreviations object to two keys for consistency with the translation files, but the client is expected to perform some sort of locale matching. Even if we were to set `pt`, the environment’s locale may be something else like `pt-AO` or even `pt-US`. There are plenty of libraries that can perform locale matching, such as [locale-utils](https://github.com/mapbox/locale-utils/).

I’m not necessarily opposed to setting the language-only locale, but I think we’d want to do so consistently for all languages and resource types, and I’m not sure that would be feasible for a situation like `zh`.

Reply:
Actually I don't see the problem with this grammar use for all Portuguese dialects - grammar expressions will match only Portuguese street names even if `pt-US` will be used in US with English names. Or there is a difference in Portuguese articles usage in`pt-BR` and `pt-US`?

Reply:
Right, I think abbreviations and grammar rules should be available regardless of the specific Portuguese locale in use, but it should be up to the client to choose a country to default to. For now the two locales share the same abbreviations and grammar rules, but that wouldn’t necessarily be the case in the future for all languages, so I’d be hesitant to create an expectation that clients can look up grammars without performing locale matching first, which they have to do when getting a translated instruction.

If it’s a major inconvenience for clients to perform locale matching themselves, then we could have this library depend on locale-utils, but there are larger libraries with more robust locale matching and I wouldn’t want to force clients to use the more rudimentary logic in locale-utils.